api_documentation.md

```diff
@@ -0,0 +1,57 @@
+# API Documentation
+
+> Note: all endpoints return JSON
+
+### /api/login
+*method:* **POST**  
+*parameters:* **username**, **sim_id**, **password**  
+*description:* creates a user entry in the database and compares the entered password with the simulation password
+*onsuccess returns:* the user model  
+
+### /api/simulations
+*method:* **GET**  
+*parameters:* **none**  
+*description:* fetches all the simulations
+*onsuccess returns:* list of simulations in the database  
```

Comment:
Yeah, my markdown preview is different than github's, so it's different sometimes! it'll be fixed in the next commit